app/controllers/advertisements_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AdvertisementsController < ApplicationController
+  before_action :set_advertisement, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @advertisements = Advertisement.all
+  end
+
+  def show
+  end
+
+  def new
+    @advertisement = Advertisement.new
+  end
+
+  def edit
+  end
+
+  def create
+    @advertisement = Advertisement.new(advertisement_params)
+    @advertisement.user = current_user
+
+    if @advertisement.save
+      redirect_to @advertisement,
+        notice: 'Advertisement was successfully created.'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.